person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
+
+# person = Person.new(name: 'ben', age: 34, parent_permission: true)
+
+# puts person.can_use_services?
```

Comment:
- hey, let's kindly remove these commented pieces of code to make our codebase clean and maintainable.